community/terraform/APKBUILD

```diff
@@ -21,12 +21,13 @@ prepare() {
 
 build() {
 	cd "$builddir"
-	GOPATH="$srcdir" go build -v -o bin/$pkgname
+	GOPATH="$srcdir" go build -v -o bin/$pkgname \
+		-ldflags "-X main.GitCommit=v$pkgver -X github.com/hashicorp/terraform/version.Prerelease= -s -w"
 }
 
 check() {
 	cd "$builddir"
-	GOPATH="$srcdir" go test
```

Comment:
Why change this? I thought `check()` is to run the test suite, not "Hello World"?

Reply:
Fixed